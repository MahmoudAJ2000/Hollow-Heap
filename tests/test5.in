5 15 5 4
2 4 635
1 0 325
4 1 951
3 2 658
2 1 215
3 0 452
0 1 738
4 0 109
4 2 802
1 2 928
0 2 709
1 3 951
3 4 986
1 4 969
2 3 508
4
2
3
0
1
