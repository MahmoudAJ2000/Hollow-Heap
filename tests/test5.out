Distance from 4 to 4 is 0
Distance from 4 to 2 is 802
Distance from 4 to 3 is 1310
Distance from 4 to 0 is 109
Distance from 4 to 1 is 847
